### ISSUE TITLE
Add to documentation used connections

### ISSUE THREAD

**Issue by VitorTrin** [mbuhot/ecto_job#61]
Our application was using more connections than what we configured for our Repo. We later found out that every job queue uses an exclusive connection, and that caused the strange number of connections.

Somewhere in the documentation it should explained that usage of connections so the user can understand the behaviour of his application.

**Comment by mbuhot** [comment]
Yes the `Postgrex.Notifications` listener consumes a DB connection that isn't from the Repo pool.

Where would you expect to see such information? Perhaps in the https://github.com/mbuhot/ecto_job#how-it-works section of the readme?

**Comment by VitorTrin** [comment]
Yeah, it seems to be the ideal place.
